Data/Text/Lazy/Builder/Int.hs

```diff
@@ -38,19 +38,9 @@ import Prelude hiding ((<>))
 #ifdef  __GLASGOW_HASKELL__
 # if defined(INTEGER_GMP)
 import GHC.Integer.GMP.Internals (Integer(S#))
-# elif defined(INTEGER_SIMPLE)
-import GHC.Integer
-# else
-# error "You need to use either GMP or integer-simple."
```

Comment:
where did this error go to?

Reply:
Not required as we only need to know whether gmp-specific things shall be included (the `S#` constructor)